### PR TITLE
allow single AtomMapper to be passed to network planners

### PR DESCRIPTION
**Pull request by richardjgowers**

### CODE REVIEW

File: openfe/setup/ligand_network_planning.py [hunk 1]
```diff
@@ -49,6 +50,8 @@ def generate_radial_network(ligands: Iterable[SmallMoleculeComponent],
       If no scorer is supplied, the first mapping provided by the iterable
       of mappers will be used.
     """
+    if isinstance(mappers, AtomMapper):
```
Comment (IAlibay) [inline openfe/setup/ligand_network_planning.py:53]: having to repeat this is pretty ugly :/ could we not wrap it up into a decorator?

Comment (richardjgowers) [reply openfe/setup/ligand_network_planning.py:53]: nah, it makes it hard to read (the function is in two places).  It's two lines, it's fine

Comment (dwhswenson) [reply openfe/setup/ligand_network_planning.py:53]: Again, a call for Konnektor (or however the German spells it), where we could put the real work in a private method and the public method could handle this.

File: openfe/tests/setup/test_network_planning.py [hunk 2]
```diff
@@ -133,6 +138,29 @@ def scoring_func(mapping):
...
+    def scorer(mapping):
+        return 1.0 / len(mapping.componentA_to_componentB)
+
+    network = openfe.ligand_network_planning.generate_minimal_spanning_network(
+        ligands=ligands,
+        mappers=mappers,
+        scorer=scorer,
+    )
+
+    assert isinstance(network, openfe.LigandNetwork)
```
Comment (dwhswenson) [inline openfe/tests/setup/test_network_planning.py:161]: Add an assertion on the number of edges? (can be a `MultiDiGraph`, but shouldn't actually be multi).

**Review by dwhswenson — APPROVED** [review]
Minor changes suggested. Otherwise good by me.

**Pull request merged**